cabal.project

```diff
@@ -22,16 +22,18 @@ source-repository-package
 
 index-state:
   -- Bump this if you need newer packages from Hackage
-  , hackage.haskell.org 2024-06-06T00:00:00Z
+  , hackage.haskell.org 2024-07-23T00:03:37Z
   -- Bump this if you need newer packages from CHaP
-  , cardano-haskell-packages 2024-06-18T00:00:00Z
+  , cardano-haskell-packages 2024-07-23T08:58:13Z
+
 
 packages:
   eras/allegra/impl
   eras/alonzo/impl
   eras/alonzo/test-suite
   eras/babbage/impl
   eras/babbage/test-suite
+  eras/byron/crypto
```

Comment:
This is already mentioned in this list, a [few lines below](https://github.com/IntersectMBO/cardano-ledger/pull/4451/files#diff-b8ed06757045fac949c89f2139a862498ad2b6d1f82c61a31e7c91d6cf0eaa70L61)

Is it on purpose  duplicated here?

Reply:
```suggestion
```